docs/reference/commandline/network_create.md

```diff
@@ -120,6 +121,11 @@ docker network create -d overlay
 ```
 Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
 
+### Network internal mode
+
+By default, when you connect a container to an `overlay` network, docker also connects a bridge network to it to provide external connectivity.
+If you want to create an externally isolated `overlay` network, you can specify the `--internal` option.
```

Comment:
tl;dr; I see there was some discussion about the flag already, just wondering: if a flag is only supported by a single driver, should it be a top-level flag?